vendor/bat-native-confirmations/src/bat/confirmations/internal/confirmations_create_confirmation_request_unittest.cc

```diff
@@ -96,15 +99,19 @@ TEST_F(ConfirmationsCreateConfirmationRequestTest, BuildBody_Viewed) {
 
   ConfirmationInfo info;
   info.creative_instance_id = creative_instance_id;
-  info.blinded_payment_token = blinded_token;
   info.type = ConfirmationType::kViewed;
-  auto payload = request_->CreateConfirmationRequestDTO(info);
+  info.blinded_payment_token = blinded_token;
+  info.country_code = "US";
+
+  const std::string platform = PlatformHelper::GetInstance()->GetPlatformName();
+  auto payload =
+      request_->CreateConfirmationRequestDTO(info, "release", platform);
 
   // Act
   auto body = request_->BuildBody(payload);
 
   // Assert
-  std::string expected_body = R"({"blindedPaymentToken":"PI3lFqpGVFKz4TH5yEwXI3R/QntmTpUgeBaK+STiBx8=","creativeInstanceId":"546fe7b0-5047-4f28-a11c-81f14edcf0f6","payload":{},"type":"view"})";  // NOLINT
+  std::string expected_body = base::StringPrintf("{\"blindedPaymentToken\":\"PI3lFqpGVFKz4TH5yEwXI3R/QntmTpUgeBaK+STiBx8=\",\"channelName\":\"release\",\"countryCode\":\"US\",\"creativeInstanceId\":\"546fe7b0-5047-4f28-a11c-81f14edcf0f6\",\"payload\":{},\"platform\":\"%s\",\"type\":\"view\"}", platform.c_str());  // NOLINT
```

Comment:
just curious, why use NO_LINT? It can be pretty-formatted using raw c++ literals

Reply:
Raised https://github.com/brave/brave-browser/issues/9167 to fix across Brave Ads